### PR TITLE
Add API ref for new endpoints

### PR DESCRIPTION
**Pull request by lovrocolic**

### CODE REVIEW

File: docs/api/10_invoices/get-all-invoices.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+sidebar_position: 8
```
Comment (sarkissianraffi) [inline docs/api/10_invoices/get-all-invoices.mdx:2]: Sidebar position should be `1`

File: docs/api/10_invoices/get-invoice.mdx [hunk 2]
```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 7
```
Comment (sarkissianraffi) [inline docs/api/10_invoices/get-invoice.mdx:2]: Sidebar position should be `2`

File: docs/api/13_organizations/update-organization.mdx [hunk 3]
```diff
@@ -0,0 +1,226 @@
+---
+sidebar_position: 3
```
Comment (sarkissianraffi) [inline docs/api/13_organizations/update-organization.mdx:2]: Sidebar position should be `2`

**Review by sarkissianraffi — CHANGES_REQUESTED** [review]
Sidebar positions need to be reviewed. Nothing blocking the release but could be messy. All endpoints seem fine to me.

**Comment by lovrocolic** [comment]
> Sidebar positions need to be reviewed. Nothing blocking the release but could be messy. All endpoints seem fine to me.

As discussed offline, final version in this branch is correct so I'm resolving comments

**Review by sarkissianraffi — APPROVED** [review]

**Pull request merged**